importer/tasks.py

```diff
@@ -263,7 +263,7 @@ def import_item_count_from_url(import_url):
         resp.raise_for_status()
         item_data = resp.json()
         output = len(item_data["resources"][0]["files"])
-        return f"{import_url} {output}", output
+        return f"{import_url} - Asset Count: {output}", output
 
     except Exception as exc:
         return f"{exc}", 0
```

Comment:
In addition to logging it, I would add something to prefix the exception since it's not always immediately obvious from the string value. Something like `f"Unhandled exception importing {import_url}: {exc}"`

Reply:
This is the message that the user sees - formatted.

Reply:
Yes – the exception path should match that behaviour